src/components/ui/calendar.tsx

```
@@ -0,0 +1,211 @@
+        Root: ({ className, rootRef, ...props }) => {
+          return (
+            <div
+              data-slot="calendar"
+              ref={rootRef}
+              className={cn(className)}
+              {...props}
+            />
+          )
+        },
+        Chevron: ({ className, orientation, ...props }) => {
+          if (orientation === "left") {
```

ESLint

[error] Replace `"left"` with `'left'`

(prettier/prettier)